helm/adelphi/templates/cassandra-image-builder.yaml

```diff
@@ -10,20 +10,33 @@ spec:
   - name: cassandra-image-template
     inputs:
       parameters:
-      - name: registry_ip
       - name: git_identifier
     container:
-      image: gcr.io/kaniko-project/executor:v1.0.0
+      image: gcr.io/kaniko-project/executor:v1.6.0-debug
```

Comment:
The general-purpose Kaniko image extends Docker's scratch image, which is a really bare system with no networking utilities and no way to install them. Fortunately, the Kaniko project has this `-debug` version that extends busybox which at least has `wget` installed. We need wget to check if the sidecar proxy is already running  before running the Kaniko executor, otherwise it will fail immediately.